<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="tomek-OptiPlex-790" ignored="0" name="Test" tests="2" failures="1" timestamp="26 lip 2017 10:31:47 GMT" time="222.211" errors="0">
  <testcase name="a" time="0.51" classname="testv6.NewTest">
    <failure type="java.lang.AssertionError" message="expected [2] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [0]
	at testv6.NewTest.a(NewTest.java:35)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- a -->
  <testcase name="f" time="133.183" classname="testv6.NewTest"/>
</testsuite> <!-- Test -->
